beacon-chain/rpc/prysm/v1alpha1/validator/proposer_deneb.go

```diff
@@ -56,15 +57,19 @@ func (c *blobsBundleCache) prune(minSlot primitives.Slot) {
 }
 
 // buildBlobSidecars given a block, builds the blob sidecars for the block.
-func buildBlobSidecars(blk interfaces.SignedBeaconBlock) ([]*ethpb.BlobSidecar, error) {
+func buildBlobSidecars(blk interfaces.SignedBeaconBlock, blobs [][]byte, kzgproofs [][]byte) ([]*ethpb.BlobSidecar, error) {
```

Comment:
Maybe I should abstract these arguments for blobs and kzgproofs so they don't accidently get swapped?